modules/nlu/src/backend/module-lifecycle/on-server-ready.ts

```diff
@@ -25,10 +25,11 @@ export function getOnServerReady(state: NLUState) {
     }
 
     const cancelTraining = async (botId: string, language: string) => {
-      const trainSession: TrainingSession = _.get(state, `nluByBot.${botId}.trainSessions.${language}`)
+      const trainSession: sdk.NLU.TrainingSession = _.get(state, `nluByBot.${botId}.trainSessions.${language}`)
 
       if (trainSession && trainSession.status === 'training') {
         if (trainSession.lock) {
+          // tslint:disable-next-line: no-floating-promises
```

Comment:
I know I'm the one who wrote that piece of code but I can't seem to remember why we don't `await` this promise

Reply:
can't think of a reason.. next time we add a `no-floating` let's add a comment that explains why.  it's scary to remove otherwise.  let's `await` it instead of adding a no-floating

Reply:
can this break anything?